cmk/special_agents/agent_kubernetes.py

```diff
@@ -949,7 +950,7 @@ def pods_per_node(self):
         return {
             node: {
                 'requests': {
-                    'pods': len(list(pods))
+                    'pods': len(list(filter(lambda pod: pod.phase == "Running", pods)))
```

Comment:
Can you do it the other way round and use a list comprehension as suggested by pylint, i.e.:

`len([pod for pod in pods if pod.status not in ["Succeeded", "Failed"]])`

This way we'll count "Pending" and "Unknown" pods as well.

The same filter will be needed in the function `pods_in_cluster` below. Otherwise the numbers for the cluster and the nodes will be inconsistent.